### ISSUE TITLE
Holidays Calendar en Clock applet

### ISSUE THREAD

**Issue by perberos** [perberos/Mate-Desktop-Environment#20]
Morion-Self me ha acercado una idea, y codigo fuente, donde se obtiene por medio de RSS los dias "Holidays" y los muestra en la misma ventana del calendario. Me dijo que lo habia visto en KDE y le habia gustado.
Yo le he dicho que tal vez podriamos usar la base de datos de holiday de KDE, para tenerlo local.
Y que yo recordaba que el clock applet de gnome, muestra las fechas si evolution está presente. Y que posiblemente, algo de codigo ya habia sido escrito.

Pero antes que cualquier cosa, tendriamos que diseñar e investigar que se ha hecho en el applet de KDE.

**Comment by perberos** [comment]
web resource
http://www.icalworld.com/holidays.html

**Comment by ghost** [comment]
Yes, I'm working on it. I found KDE database and use it. The file-parser is almost ready (I use xml format), I'm working on the support of several countries and the interface.
While only be displayed near to the calendar, the backlight dates on the calendar will later.
